src/debug/createdump/crashinfo.cpp

```diff
@@ -708,6 +708,17 @@ CrashInfo::EnumerateManagedModules(IXCLRDataProcess* pClrDataProcess)
             break;
         }
 
+        // Skip any dynamic modules. The Request call below on some DACs crashes on dynamic modules.
```

Comment:
Do we understand the impact of skipping the dynamic modules? Are there other things that will start failing  downstream when we have no data for dynamic modules?